sdk/core/azure-core/inc/azure/core/http/curl/curl.hpp

```diff
@@ -31,7 +31,7 @@ namespace Azure { namespace Core { namespace Http {
      * check by default.
      *
      */
-    bool NoRevoke = true;
+    bool DisableCertificateRevocationListCheck = true;
```

Comment:
nit: It's nice to have default values be the same as "0", so `false` for bool. This is quite common in .NET:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/default-values

In C++, default values of bool is false, as well.

So, consider naming it Enable rather than Disable, and negate it.

```suggestion
    bool EnableCertificateRevocationListCheck = false;
```

Reply:
The default value from libcurl here is `false`, We mention that in the comments. Then we mention how we are explicitly changing this for the SDK.

Reply:
>The default value from libcurl here is `false`, We mention that in the comments. Then we mention how we are explicitly changing this for the SDK.

I am aware of that :)

As a customer of the SDK, it doesn't matter what the libcurl defaults are. I recommend decoupling implementation detail from the public API surface. The doc comment as-is continues to hold true, even with my suggestion.

Reply:
I will play the old and hated classic quote:  "this PR is following the requested change in the issue", please approve and create a new issue if the requirement has changed.

Let's keep requirements clear and in the issue only and avoid updating the expectation on PRs.

Reply:
> "this PR is following the requested change in the issue"

I quoted the issue above. It explicitly indicated that we would want to flip disable -> enable based on the default value change (at the time, the default value was `true`). The primary goal of the issue was to avoid double negative, and we are still somewhat in that came.

I don't see this as a change of requirements or going against the issue at all.

That said, I'll leave it up to you to decide to take the suggestion, if you feel strongly about keeping it `disable`, and leaving the default value as `false`.